### ISSUE TITLE
`change` prop is unbound

### ISSUE THREAD

**Issue by almassapargali** [erikras/redux-form#2663]
### Are you submitting a **bug report** or a **feature request**?
bug report

### What is the current behavior?
In a `handleSubmit` function which looks like:
```js
handleSubmit(data, dispatch, { updateAccount, change, untouch }) {
  return updateAccount(data)
    .then(() => {
      dispatch(change("password", null));
      dispatch(change("current_password", null));
      untouch("password", "current_password");
    })
    .catch(parseFormErrors);
}
```
I had to dispatch `change` function in order to actually fire it. Just calling `change("current_password", null)` does nothing, moreover while [docs](http://redux-form.com/6.5.0/docs/api/Props.md/) stats that `change` is a bound function which returns nothing, `console.log(change("password", null))` outputs:
```
Object
  meta: {form: "AccountForm", field: "password", touch: false, persistentSubmitErrors: false}
  payload: null
  type: "@@redux-form/CHANGE"
```

### What is the expected behavior?
`change` should be bound, so I can just call it to fire an action, like how I do with `untouch`.

### What's your environment?
```
"redux-form": "^6.5.0"
Node: v7.4.0
Mac OS latest, with latest versions of Firefox & Safari
```

**Comment by erikras** [comment]
Wow, this is an interesting one. `change` _is_ bound to dispatch before it is passed into your decorated form component, but it has not yet been bound in the `Form` higher order component when it is passing `props` to the submit function.

For example, if you did this, it would work:
```jsx
class MyForm extends React.Component {
  constructor() {
    super()
    this.submit = this.submit.bind(this)
  }

  submit(data) {
    // change is bound in this.props, but not in the props given as a param
    const { updateAccount, change, untouch } = this.props
    return updateAccount(data)
      .then(() => {
        change('password', null)
        change('current_password', null)
        untouch('password', 'current_password')
      })
  }

  render() {
    const { handleSubmit } = this.props
    return <form onSubmit={handleSubmit(this.submit)}> ... </form>
  }
}
```
The `props` given as a parameter to your submit function is mainly for allowing the container component of your form to pass in some additional information, e.g. what path to forward to after submit.

**Comment by almassapargali** [comment]
Ok, got it, thanks for quick fix!

**Comment by erikras** [comment]
Fix published in [`v6.6.0`](https://github.com/erikras/redux-form/releases/tag/v6.6.0).
